examples/official-storybook/package.json

```diff
@@ -58,7 +58,7 @@
     "webpack": "4"
   },
   "peerDependencies": {
-    "puppeteer": "^2.0.0 || ^3.0.0"
```

Comment:
I think for our examples we'll want to keep using the exact same version of puppeteer we're using now, there' no need to change this.